2.0/config/settings.py

```
@@ -0,0 +1,44 @@
+TARGET_FILING_TYPES = ['10-q', '10-k', '8-k', 's-1', 's-3', 'def 14a', 'sec staff action', '13f-hr', '13f-nt', 'sc 13d', 'sc 13g']
+
+SEC_RATE_LIMIT = 10 # Req/s
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)